lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -1114,6 +1114,7 @@ def install_puppet_agent_pe_promoted_repo_on( hosts, opts )
               if arch == 'x86_64'
                 arch = 'amd64'
               end
+              version = version[0,2] + '.' + version[2,2] if variant =~ /ubuntu/
```

Comment:
Should maybe use the same solution that we have for osx?

https://github.com/puppetlabs/beaker/blob/master/lib/beaker/platform.rb#L26

Same thing, need to ensure that 1010 -> 10.10.